doc/fvwm3_manpage_source.adoc

```diff
@@ -1650,6 +1650,7 @@ $[math.+.<x>,<y>] $[math.-.<x>,<y>] $[math.*.<x>,<y>] $[math./.<x>,<y>] $[math.%
 	is replaced with 21.
 +
 The comma in "<x>,<y>" is to allow for a future possibility of using floats.
+Maximum (>) returns the greater of two values, and minimum (<) the lesser.
 All operations use C integer mathematics. The result of integer division is
```

Comment:
Move this up to the list of all the other operations in the previous pargraph.  The statement is out of place here in talking about floats and integer math.